cmd/apiserver/main.go

```diff
@@ -1,29 +1,27 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
-	apiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/component-base/cli"
 	"k8s.io/component-base/logs"
 
+	apiserver "k8s.io/apiserver/pkg/server"
+
 	"github.com/clusterpedia-io/clusterpedia/cmd/apiserver/app"
+	"github.com/spf13/cobra"
 )
 
 func main() {
-	if err := runApiServerCommand(); err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
-		os.Exit(1)
-	}
+	command := runApiServerCommand()
+	code := cli.Run(command)
+	os.Exit(code)
 }
 
-func runApiServerCommand() error {
+func runApiServerCommand() *cobra.Command {
```

Comment:
As we used `k8s.io/component-base/cli`, we don't need `runApiServerCommand()` anymore.